cherry_picker/cherry_picker/cherry_picker.py

```diff
@@ -97,8 +117,10 @@ def get_pr_url(self, base_branch, head_branch):
 
     def fetch_upstream(self):
         """ git fetch <upstream> """
+        set_state('FETCHING_UPSTREAM')
```

Comment:
Would be nice to collect all these constants in the single `enum.Enum`

Reply:
I was thinking about it, sure I can go for Enum.